src/platforms/rcore_desktop_glfw.c

```diff
@@ -144,6 +144,15 @@ bool WindowShouldClose(void)
 // Toggle fullscreen mode
 void ToggleFullscreen(void)
 {
+    if (IsWindowState(FLAG_BORDERLESS_WINDOWED_MODE))
+    {
+        // We can't go straighforward from one fullscreen mode to an other
+        // there needs to be at least one loop between them
+        // so we just toggle the currently active fullscreen mode and leave.
+        ToggleBorderlessWindowed();
```

Comment:
We could add a flag here like `fullscreenChangePending = true;` and in the main loop if this flag is true fullscreen is activated & `fullscreenChangePending = false`; (the same for borderless) 

This is not important right now I just wanted to add it for later ;)